.eslintrc

```diff
@@ -0,0 +1,78 @@
+{
+  "parser": "babel-eslint",
```

Comment:
There is a separate issue for that: https://github.com/callstack-io/react-native-material-palette/issues/29